lexicon/tests/providers/test_dreamhost.py

```diff
@@ -0,0 +1,27 @@
+"""Integration tests for Linode"""
+from unittest import TestCase
+
+import pytest
+from lexicon.tests.providers.integration_tests import IntegrationTests
+
+
+# Hook into testing framework by inheriting unittest.TestCase and reuse
+# the tests which *each and every* implementation of the interface must
+# pass, by inheritance from integration_tests.IntegrationTests
+class DreamhostProviderTests(TestCase, IntegrationTests):
+    """TestCase for Linode"""
+    provider_name = 'dreamhost'
+    domain = 'lexicon-example.com'
+
+    def _filter_post_data_parameters(self):
+        return []
+
+    def _filter_headers(self):
```

Comment:
You can remove this method if there is not header to remove.

Reply:
I'll remove this method.